src/Uno.Templates/content/unoapp/MyExtensionsApp.1.MauiControls/MyExtensionsApp.1.MauiControls.csproj

```
@@ -22,7 +22,7 @@
     </TargetFrameworks>
 
     <UseMaui>true</UseMaui>
-    <SingleProject>true</SingleProject>
+    <UnoSingleProject>true</UnoSingleProject>
```

This project doesn't use the Uno.Sdk
```suggestion
    <SingleProject>true</SingleProject>
```